tests/security/selinux/audit2allow.pm

```diff
@@ -45,8 +46,9 @@ sub run {
     upload_logs($audit_log_short);
 
     # create an SELinux module, make this policy package active, check the new added module
+    $cmd = "cat " . (is_sle('=15-SP3') ? "$original_audit" : "$audit_log_short") . "| audit2allow -M $test_module";
```

Comment:
Add one `" "` (blank) before `"|"` is better.

Reply:
good point :) I'll do this